python/singa/data.py

```diff
@@ -97,7 +101,7 @@ def start(self):
         self.p.start()
         return
 
-    def next(self):
+    def __next__(self):
```

Comment:
python-future handles this:

http://python-future.org/what_else.html#custom-iterators

There should be no errors. If you find an error, please report it.